svcs/amigo/table.go

```diff
@@ -33,40 +36,47 @@ type TeamRow struct {
 }
 
 type Scoreboard struct {
-	Chals 	[]ChalRow	`json:"challenges"`
+	Chals 	[]ChalCategory	`json:"challenges"`
```

Comment:
Naming of variables is a little  bit  confusing. Do  not you think that having following make much sense ?

```suggestion
	ChalCats 	[]ChalCategory	`json:"chal-categories"`
```

Reply:
Yeah i agree. I am going to make some comments on them